packages/lodestar-cli/src/cmds/beacon/cmds/init/init.ts

```diff
@@ -1,14 +1,26 @@
 import {Arguments} from "yargs";
+import deepmerge from "deepmerge";
 
+import {rootDir} from "../../../../options";
 import {IBeaconArgs} from "../../options";
 import {mkdir} from "../../../../util";
 import {initPeerId, initEnr, readPeerId} from "../../../../network";
 import {initBeaconConfig} from "../../config";
+import {getTestnetConfig, downloadGenesisFile} from "../../testnets";
 
 /**
  * Initialize lodestar-cli with an on-disk configuration
  */
 export async function init(args: Arguments<IBeaconArgs>): Promise<void> {
+  // Auto-setup altona
+  if (args.altona) {
```

Comment:
Might be better to use --network <altona|mainnet> as other clients

Reply:
I like the suggestion but we are already using a `network` parameter here
https://github.com/ChainSafe/lodestar/blob/9c5fef9c83645e1c26e5fb43ec263237ca0c53fb/packages/lodestar-cli/src/cmds/beacon/options/beaconFile.ts#L8
Would you suggest to use a different name than `network` that's not used or go with altona?

Reply:
argh, maybe chain.network then?

Reply:
`--testnet`? blank == mainnet